packages/desktop-file-utils/SOURCES/update-desktop-database-wrapper.sh

```diff
@@ -35,6 +35,8 @@ for DESKTOP_FILE_PATH in $(find $SGUG_DATADIR_APPS/*.desktop -type f); do
 	cp /usr/sgug/share/desktop-file-utils/sgugenv.sh $CATALOG_ROOT/$APPNAME
 
 	chmod 755 $CATALOG_ROOT/$APPNAME
-	echo "exec $DESKTOP_KEY_EXEC" >> $CATALOG_ROOT/$APPNAME
+	echo "#!/usr/sgug/bin/bash" >> $CATALOG_ROOT/$APPNAME
+	echo "export PATH=$PATH:/usr/sgug/bin" >> $CATALOG_ROOT/$APPNAME
```

Comment:
I think maybe this should be `PATH=/usr/sgug/bin:$PATH` to be sure we hit SGUG binaries before system ones.

Reply:
Yes!

Looks like I also forgot that `cp` in the spec.